src/vcSettingsUI.cpp

```diff
@@ -49,6 +49,13 @@ enum vcLASClassifications
   vcLASClassifications_LastClassification = 255 // Because they are always stored in a uint8
 };
 
+void vcSettingsUI_Cleanup(vcState *pProgramState)
+{
+  for (size_t i = 0; i < udLengthOf(s_mapTiles); i++)
```

Comment:
For what it's worth you could do a range-for loop here. `for (auto &mapTile : s_mapTiles)` or with the correct type.